src/main/java/ch/njol/skript/expressions/ExprFuseDuration.java

```diff
@@ -0,0 +1,157 @@
+		max = parseResult.hasTag("max");
+		return super.init(exprs, matchedPattern, isDelayed, parseResult);
+	}
+
+	@Override
+	public @Nullable Timespan convert(Entity entity) {
+		if (entity instanceof TNTPrimed) {
+			return getTimespan(((TNTPrimed) entity).getFuseTicks());
+		} else if (entity instanceof Creeper) {
+			return getTimespan(max ? ((Creeper) entity).getMaxFuseTicks() : ((Creeper) entity).getFuseTicks());
+		} else if (entity instanceof ExplosiveMinecart) {
+			return getTimespan(((ExplosiveMinecart) entity).getFuseTicks());
+		}
+		return null;
+	}
+
+	private Timespan getTimespan(int ticks) {
+		if (ticks < 0) {
+			ticks = 0;
+		}
+		return new Timespan(Timespan.TimePeriod.TICK, ticks);
+	}
+
+
+	@Override
+	public Class<?>[] acceptChange(Changer.ChangeMode mode) {
+		if (mode == ChangeMode.SET || mode == ChangeMode.ADD || mode == ChangeMode.REMOVE || mode == ChangeMode.DELETE || mode == ChangeMode.RESET) {
+			return CollectionUtils.array(Timespan.class);
+		}
+		return null;
+	}
+
+	@Override
+	public void change(Event event, Object @NotNull [] delta, ChangeMode mode) {
+		for (Entity entity : getExpr().getArray(event)) {
+			if (entity instanceof TNTPrimed) {
+				changeTNTFuseTicks((TNTPrimed) entity, delta, mode);
+			} else if (entity instanceof Creeper) {
+				changeCreeperFuseTicks((Creeper) entity, delta, mode);
+			} else if (entity instanceof ExplosiveMinecart) {
+				changeMinecartFuseTicks((ExplosiveMinecart) entity, delta, mode);
+			}
+		}
+	}
+
+	private void changeTNTFuseTicks(TNTPrimed tnt, Object[] delta, ChangeMode mode) {
+		int currentTicks = tnt.getFuseTicks();
+		int newTicks = calculateNewTicks(tnt, currentTicks, delta, mode);
+		tnt.setFuseTicks(newTicks);
+	}
+
+	private void changeCreeperFuseTicks(Creeper creeper, Object[] delta, ChangeMode mode) {
+		if (max) {
+			int currentMaxTicks = creeper.getMaxFuseTicks();
+			int newMaxTicks = calculateNewTicks(creeper, currentMaxTicks, delta, mode);
+			creeper.setMaxFuseTicks(Math.max(newMaxTicks, 0));
+		}
+
+		int currentTicks = creeper.getFuseTicks();
+		int maxTicks = creeper.getMaxFuseTicks();
+		int newTicks = calculateNewTicks(creeper, currentTicks, delta, mode);
+
+		creeper.setFuseTicks(Math.max(Math.min(newTicks, maxTicks), 0)); // Ensure non-negative value
+	}
+
+	private void changeMinecartFuseTicks(ExplosiveMinecart minecart, Object[] delta, ChangeMode mode) {
+		int currentTicks = minecart.getFuseTicks();
+		int newTicks = calculateNewTicks(minecart, currentTicks, delta, mode);
+		minecart.setFuseTicks(newTicks);
+	}
+
+	private int calculateNewTicks(Entity entity, int currentTicks, Object[] delta, ChangeMode mode) {
+		long deltaTicks = delta != null && delta.length > 0 ? ((Timespan) delta[0]).getAs(Timespan.TimePeriod.TICK) : 0;
+		long newTicks = switch (mode) {
+			case SET -> deltaTicks;
+			case ADD -> (long) currentTicks + deltaTicks;
+			case REMOVE -> (long) currentTicks - deltaTicks;
```

Comment:
I would use Math2.addClamped here in cases of Long overflow. You can just make `deltaTicks` negative for `remove`